json-schemas/src/client-events-api-requests.json

```diff
@@ -5,6 +5,10 @@
   "description": "A message sent on the clientEvents:apiRequests metachannel describing a sampled client API request.",
   "properties": {
     "host": {
+      "type": "string",
+      "description": "The HTTP Method of the API request."
```

Comment:
I think the descriptions are in the wrong place, this is for `host`.

Reply:
Thanks - I swapped them now.